charts/appgate-server/values.yaml

```diff
@@ -20,8 +20,12 @@ config:
   signing_key: app1
   # The host and port that the appgate server will listen on
   listening_endpoint: http://0.0.0.0:42069
-  # tcp://<host>:<port> to a full pocket node for reading data and listening for on-chain events
-  query_node_url: tcp://poktroll-sequencer:36657
+  # tcp://<host>:<port> to a full pocket node for reading data
+  query_node_grpc_url: tcp://poktroll-sequencer:36658
+  # grpc client insecure flag
+  grpc_insecure: true
+  # tcp://<host>:<port> to a full pocket node websocket endpoint listening for on-chain events
+  pocket_node_websocket_url: tcp://poktroll-sequencer:36657
```

Comment:
We have `query_node_grpc_url` and `pocket_node_websocket_url`. 

I feel like `query_node_grpc_url` and `query_node_websocket_url` would be more appropriate given that the comments are saying that they're both for reading / listening for on-chain data. You can "query" data via poll or listening based mechanisms.

I realize the comments may not be reflected in how it's actually being used so wanted to discuss the naming before we move forward.

Reply:
So we'll have `query_node_grpc_url`, `query_node_websocket_url` and `network_node_grpc_url` right?

If that's the case, I'm OK with it and will reflect these changes here and to Pocktroll corresponding PR

Reply:
Rough Notes from call

---

```bash
# Pocket node URL that exposes CometBFT JSON-RPC API.
# This can be used by the cosmos client SDK, event subscriptions, etc...
`query_node_rpc_url`

# Pocket node URL that exposes the Cosmos gRPC service.
`tx_node_grpc_url`

# Pocket node URL that exposes the Cosmos gRPC service, dedicated to querying purposes.
# If unspecified, defaults to `tx_node_grpc_url`.
`query_node_grpc_url`
```

--- 

# Context:
Cosmos Client:
- QueryClient
	- QueryNodeURL
- TxClient
    - TxNodeURL
   
## Tentative changes 1 
Changes if we go with path:    
1. Rename `NetworkNodeUrl` to `TxNodeUrl` so that naming is consistent
2. Update comments to explain that `QueryUrl` is for `CosmosQueryClient` and `TxUrl` is for `CosmosTxClient`

Questions:
- Why are we even doing this?
- What are we trying to achieve?
- Should we even switch to gRPC?
- Are we even switching to something more complex?

What is the problem we're solving?
- `Goal`: Import the SDK without requiring an HTTP server
   - Path to achieve to goal: import SDK w/o running an external binary
- `Problem`: SDK is dependant on cosmos context
- `Solution`: Refactoring the SDK to build a gRPC client instead of a cosmos client

- `Original problem`:
   - Prevent the SDK from being dependent on the cosmos SDK content
   - Start `AppGateServer` and `RelayMiner` independent of `controlled binary`

Some motivation:
- Enable SDK to use gRPC
- Continue using the regular RPC client


Two endpoints:
1. gRPC endpoint
   1.1 Query endpoint (querying or polling)
   1.2 Tx endpoint (send or push)
2. WebSocket endpoint (event subscriptions - listening)

Requirements:
- `1` & `2` are needed at a minimum
- `1.1` can be the same as `1.2` but not a hard requirement (default to the same value if unspecified)